src/shared/components/NotFound.tsx

```diff
@@ -36,106 +37,112 @@ import {CLOUD} from 'src/shared/constants'
 // API
 import {fetchDefaultAccountDefaultOrg} from 'src/identity/apis/org'
 
-const NotFoundNew: FC = () => (
-  <AppWrapper type="funnel" className="page-not-found" testID="not-found">
-    <FunnelPage enableGraphic={true} className="page-not-found-funnel">
-      {CLOUD && (
+const NotFoundNew: FC = () => {
```

Comment:
It's a little hard to parse this diff. Is this a reversion of the previous commit switching from email?